react/features/transcription/actions.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+import {
+    ADD_TRANSCRIPT_MESSAGE,
+    ENDPOINT_MESSAGE_RECEIVED,
+    REMOVE_TRANSCRIPT_MESSAGE,
+    UPDATE_TRANSCRIPT_MESSAGE
+} from './actionTypes';
+
+/**
+ * Signals that a transcript with a new message_id is received.
+ *
+ * @param {string} transcriptMessageID - The new message_id.
+ * @param {string} participantName - The payload carried in the message.
+ * @returns {{
+ *      type: ADD_TRANSCRIPT_MESSAGE,
+ *      transcriptMessageID: string,
+ *      participantName: string
+ * }}
+ */
+export function addTranscriptMessage(transcriptMessageID: string,
+        participantName: string) {
+    return {
+        type: ADD_TRANSCRIPT_MESSAGE,
+        transcriptMessageID,
+        participantName
+    };
+}
+
+/**
+ * Signals that a participant sent an endpoint message on the data channel.
+ *
+ * @param {JitsiConference} conference - The JitsiConference which had its lock
+ * state changed.
+ * @param {Object} participant - The participant details sending the message.
+ * @param {Object} payload - The payload carried by the endpoint message.
+ * @returns {{
+ *      type: ENDPOINT_MESSAGE_RECEIVED,
+ *      conference: JitsiConference,
+ *      participant: Object,
+ *      payload: Object
+ * }}
+ */
+export function endpointMessageReceived(conference: Object,
+        participant: Object, payload: Object) {
+    return {
+        type: ENDPOINT_MESSAGE_RECEIVED,
+        conference,
```

Comment:
How is conference used? I'm missing the place where it's called.

Reply:
No, its not being used anymore. :+1: